roottest/root/io/hadd/CMakeLists.txt

```diff
@@ -1,27 +1,30 @@
-if(NOT ROOT_pyroot_FOUND)
-    return()
+if(NOT pyroot)
+  return()
 endif()
 
 ROOTTEST_ADD_TEST(input_validation
-                  MACRO input_validation.py)
+                  COPY_TO_BUILDDIR input_validation.py
+                  MACRO ${CMAKE_CURRENT_BINARY_DIR}/input_validation.py)
```

Comment:
Why this change (here and line 12)?

Reply:
When running python script in source directory - it creates `__pycache__` in source directory.
To avoid such side-effect I copied script to build directory first.